src/partials/cookie-further-settings.hbs

```diff
@@ -0,0 +1,272 @@
+<div class="row py-3" id="cookie-further-settings">
+   <div class="col-12 mb-3">
+      <div class="privacy-settings d-flex flex-column">
+         <div class="privacy-settings-body overflow-auto">
+            <div class="card consent-group">
+               <div class="card-body mb-0">
+                  <p class="card-title h4 d-flex"><span class="flex-fill">
+                     Essential
+                     (4)
+                     </span> <span class="badge badge-primary bg-appearance">Essential</span>
+                  </p>
+                  <p class="card-text">Essential cookies enable basic functions and are necessary for the proper functioning of the website.</p>
```

Comment:
You should get the translations instead of hardcoding text

Reply:
+1

I've already started working on the translations.